dask_pytorch/results.py

```diff
@@ -0,0 +1,91 @@
+"""
+Infrastructure for retrieving and logging intermediate results from pytorch training jobs.
+
+Currently using dask pub/sub, but will create an S3 version in the future.
+"""
+import uuid
+import logging
+import os
+from typing import List
+from os.path import join, exists, dirname
+
+from distributed.pubsub import Pub, Sub
+from distributed.utils import TimeoutError  # pylint: disable=redefined-builtin
```

Comment:
```suggestion
from distributed.utils import as DistributedTimeoutError
```

I think it would be cleaner and a little safer to just rename this on import, and that that's better than ignoring the `pylint` warning.